src/components/autoSuggest/index.js

```diff
@@ -260,10 +259,16 @@ class AutoSuggest extends React.Component {
       });
     return this.recentSearches;
   }
-
   render() {
     const { t } = this.props;
 
+    const getNoResultMessage = () => {
+      let noResultMessage;
+      if (this.state.value.length > 0 && this.state.value.length <= 2) noResultMessage = 'Type at least 3 characters';
+      if (this.state.value.length > 2 && this.state.resultsLength === 0) noResultMessage = 'No results found';
+      return noResultMessage;
+    };
```

Comment:
I thought `getNoResultMessage` would be a separate method outside of `render`. `render` is quite long anyway.


The code-base mostly uses `if-else` with curlies and separate lines. Please adhere to this style:
https://github.com/LiskHQ/lisk-hub/blob/81e6c9f46d12947a8cc4a5b2c024db5eb34571df/src/components/autoSuggest/index.js#L165-L169